### PR TITLE
Create RequestData header

### PR DESCRIPTION
**Pull request by will889889**
RequestData is a class of a data structure that represents a request.

### CODE REVIEW

**Comment by will889889** [comment]
Done

File: RequestData.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+//    data.port = 80;
+//    data.target = "3arUqBH";
+//    data.requestMethod = "Search";
+class RequestData
+{
+public:
+	// Default constructor, all memeber variables would be 0 or "";
+	RequestData();
+	// Constructor that sets all the memeber variables with the parameters.
+	RequestData(std::string _host, int _port, std::string _target, std::string _requestMethod);
```
Comment (destinycan) [inline RequestData.h:28]: how about use `uint16_t ` in _port

File: RequestData.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	// Default destructor
+	~RequestData();
+
+	std::string host;
+	int port;
```
Comment (destinycan) [inline RequestData.h:33]: the same as the previous comment, u can use `uint16_t

**Comment by will889889** [comment]
Done, thank you.

**Pull request merged**